boards/BMS/BMS.ioc

```diff
@@ -13,17 +13,18 @@ ADC1.EnableInjectedConversion=ENABLE
 ADC1.EnableRegularConversion=ENABLE
 ADC1.ExternalTrigConv=ADC_EXTERNALTRIGCONV_T3_TRGO
 ADC1.ExternalTrigConvEdge=ADC_EXTERNALTRIGCONVEDGE_RISING
-ADC1.IPParameters=ClockPrescaler,Resolution,DataAlign,ScanConvMode,ContinuousConvMode,DiscontinuousConvMode,DMAContinuousRequests,EOCSelection,Overrun,LowPowerAutoWait,EnableRegularConversion,NbrOfConversion,ExternalTrigConv,ExternalTrigConvEdge,EnableInjectedConversion,InjNumberOfConversion,EnableAnalogWatchDog1,EnableAnalogWatchDog2,EnableAnalogWatchDog3,Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,master,SubFamily
+ADC1.IPParameters=ClockPrescaler,Resolution,DataAlign,ScanConvMode,ContinuousConvMode,DiscontinuousConvMode,DMAContinuousRequests,EOCSelection,Overrun,LowPowerAutoWait,EnableRegularConversion,NbrOfConversion,ExternalTrigConv,ExternalTrigConvEdge,EnableInjectedConversion,InjNumberOfConversion,EnableAnalogWatchDog1,EnableAnalogWatchDog2,EnableAnalogWatchDog3,Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,master,SingleDiff-0\#ChannelRegularConversion,SubFamily
 ADC1.InjNumberOfConversion=0
 ADC1.LowPowerAutoWait=DISABLE
 ADC1.NbrOfConversion=1
-ADC1.Offset-0\#ChannelRegularConversion=0
-ADC1.OffsetNumber-0\#ChannelRegularConversion=ADC_OFFSET_NONE
+ADC1.Offset-0\#ChannelRegularConversion=1983
+ADC1.OffsetNumber-0\#ChannelRegularConversion=ADC_OFFSET_3
 ADC1.Overrun=ADC_OVR_DATA_OVERWRITTEN
 ADC1.Rank-0\#ChannelRegularConversion=1
 ADC1.Resolution=ADC_RESOLUTION_12B
-ADC1.SamplingTime-0\#ChannelRegularConversion=ADC_SAMPLETIME_61CYCLES_5
+ADC1.SamplingTime-0\#ChannelRegularConversion=ADC_SAMPLETIME_601CYCLES_5
```

Comment:
Did we try playing with the sample time after changing the adc offset to `ADC_OFFSET_3`? Might be a good idea to play around with this value

Reply:
We did not. Is it better to have lower or higher sample times from an MCU standpoint?

Reply:
![image](https://user-images.githubusercontent.com/32230980/157176542-ab6f7c04-8661-4752-bc64-ccc92ce6243f.png)

This will affect our total conversion time, better to have a lower sample time

Reply:
We may introduce more inaccuracies if we lower the sample time all the way to 1.5 ADC clock cycles. I think it's best to leave it at around 7.5 clock cycles to be safe. See [Sec 4.2.6](https://www.st.com/resource/en/application_note/cd00211314-how-to-get-the-best-adc-accuracy-in-stm32-microcontrollers-stmicroelectronics.pdf)